README.rst

```diff
@@ -0,0 +1,22 @@
+h5py_wrapper
+============
+
+A wrapper to conveniently store nested python dictionaries in hdf5 files. It exposes two basic functions to the user `wrapper.save` and `wrapper.load`.
+
+
+.. image:: https://img.shields.io/badge/python-2.6%2C%202.7-blue.svg
+   :target: www.python.org
+   :alt: Python Version
+
+	 
+Documentation
+-------------
+http://h5py-wrapper.readthedocs.io/en/latest/
+
+
+Code status
+-----------
+
+.. image:: https://travis-ci.org/INM-6/python-dicthash.svg?branch=master
```

Comment:
while your're at it, could you change this to the correct .svg file? the link is currently to a different package.